projects/cashmere-examples/src/lib/stepper-overview/stepper-overview-example.component.ts

```diff
@@ -29,4 +28,11 @@ export class StepperOverviewExampleComponent implements OnInit {
             {label: 'Year Round', iconSet: 'fa', icon: 'fa-lock', disabled: true}
         ];
     }
+
+    ngAfterViewInit() {
```

Comment:
This is interesting. From a code style perspective, most of the other Cashmere examples use parameters on the component tag, but I don't think either way is necessarily right or wrong. I'm assuming this change was just a refactor - or was their a functionality reason to define these params after view init?

Reply:
There are many times when a variable isn't needed for setting these properties and there is no need to make a property public. It works the same either way.  This is a refactor that I wanted to validate by using the routes to set the active step instead of a click or variable.

Reply:
Gotcha - that's fine by me